lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -155,6 +157,7 @@ export abstract class ClientApplication {
 
         // Initialize the token cache
         this.tokenCache = new TokenCache(this.config, this.browserStorage, this.logger, this.browserCrypto);
+        this.trackPageVisibilityWithMeas = this.trackPageVisibilityWithMeas.bind(this);
```

Comment:
WithMeas => `WithMeasurement`  Consistency is good and you're mixing Meas as an abbreviation along with spelling it out completely like you do on lines 86, 87 just above.

Reply:
fixed